backend/src/routes/items.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require("mongoose")
+const express = require("express")
+const router = express.Router()
+const ItemController = require("../controllers/items")
+
+// Get All Items
+router.get("/", ItemController.getAllItems)
+// Get Item by ID
+router.get("/:id", ItemController.getItemById)
+// Create Item
+router.post("/", ItemController.createItem)
```

Comment:
Create, Update and delete items are admin only actions. You can use the `requireRole` middleware for that.
```js
router.post("/", requireRole(roles.ADMIN, ItemController.createItem
```